File: GDocs.Site/src/app/notificacao/models/notificacao-relatorio.model.ts

export class NotificacaoRelatorioModel {

    constructor (
        public idNotificacaoUsuario: number, 
        public idRelatorio: number, 
        public idUsuario: string, 
        public lido: boolean,  
        public nome: string, 
        public url: string,
        public parametros: string,
        public rdlBytes: string
        ) 
    { }
}

File: GDocs.Api/src/Infra/CrossCutting/Models/NotificacaoRelatorioModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ICE.GDocs.Infra.CrossCutting.Models
{
    public class NotificacaoRelatorioModel : BaseModel
    {
        public int IdNotificacaoUsuario { get; set; }
        public int IdRelatorio { get; set; }
        public Guid IdUsuario { get; set; }
        public bool Lido { get; set; }
        public DateTime DataLeitura { get; set; }
        public string Nome { get; set; } = string.Empty;
        public string Url { get; set; } = string.Empty;
        public string Parametros { get; set; } = string.Empty;
        public byte[] RdlBytes { get; set; }
    }
}
